site/src/api/api.ts

```diff
@@ -1871,20 +1908,20 @@ class ApiMethods {
 		templateId: string,
 	): Promise<TypesGen.DAUsResponse> => {
 		const response = await this.axios.get(
-			`/api/v2/templates/${templateId}/daus`,
+			`${API_PREFIX}/templates/${templateId}/daus`,
 		);
 
 		return response.data;
 	};
 
 	getDeploymentDAUs = async (
 		// Default to user's local timezone.
-		// As /api/v2/insights/daus only accepts whole-number values for tz_offset
+		// As ${API_PREFIX}/insights/daus only accepts whole-number values for tz_offset
```

Comment:
nit: maybe keeping the full URL here is cleaner for readability?
```suggestion
		// As /api/v2/insights/daus only accepts whole-number values for tz_offset
```